decidim-admin/app/controllers/decidim/admin/participatory_process_steps_controller.rb

```diff
@@ -68,14 +68,25 @@ def destroy
         redirect_to participatory_process_path(@participatory_process_step.participatory_process)
       end
 
+      def ordering
+        authorize ParticipatoryProcessStep
+
+        ReorderParticipatoryProcessSteps.call(collection, params[:items_ids]) do
+          on(:invalid) do
+            flash.now[:alert] = I18n.t("participatory_process_steps.ordering.error", scope: "decidim.admin")
+            redirect_to participatory_process_path(participatory_process)
```

Comment:
We don't do anything. If I put a render or a redirect here, then page refreshes and misses all the point. 

We could render some JS here and then do something in the layout, but if we need to show a success message every time the user drags and drops... I think it's too much.